"""Test case for functions defined in try-except-else blocks."""

from test_module import get_function

# This should work - get_function should be accessible
result = get_function()
print(result)
